cluster/images/hyperkube/master.json

```diff
@@ -31,7 +31,7 @@
               "/hyperkube",
               "apiserver",
               "--service-cluster-ip-range=10.0.0.1/24",
-              "--insecure-bind-address=127.0.0.1",
+              "--insecure-bind-address=0.0.0.0",
```

Comment:
This looks unrelated? Also, doesn't this expose the insecure outside the local machine? Intentional?